apps/cowswap-frontend/src/common/pure/Modal/index.tsx

```diff
@@ -141,3 +145,169 @@ export const CowModal = styled(Modal)<{
     }
   }
 `
+
+// New Modal to be used going forward =================================
+const ModalInner = styled.div`
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  height: auto;
+  margin: auto;
+  background: var(--cow-container-bg-01);
+  border-radius: var(--cow-border-radius-normal);
+  box-shadow: var(--cow-box-shadow-normal);
```

Comment:
And here

```suggestion
  background: var(${ColorTypes.CONTAINER_BG_01});
  border-radius: var(${ColorTypes.BORDER_RADIUS_NORMAL});
  box-shadow: var(${ColorTypes.BOX_SHADOW_NORMAL});
```

And so on